composer.json

```diff
@@ -95,7 +95,8 @@
     "scheb/two-factor-bundle": "^3.26",
     "phpoffice/phpspreadsheet": "^1.9",
     "html2text/html2text": "^4.2.1",
-    "webmozarts/console-parallelization": "^1.1.0"
+    "webmozarts/console-parallelization": "^1.1.0",
+    "spiritix/php-chrome-html2pdf": "^1.5"
```

Comment:
This should be an optional requirement.

Reply:
I removed it from the composer.json

Reply:
The code should somehow check if it's installed, then. And it should be added to the composer `suggest` section, I suppose.